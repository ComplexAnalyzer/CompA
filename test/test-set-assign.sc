def main() {
    a= [1,2,3,4];
    (0 <= i < 4| true) print(a[i]); 
    print(#a);  
    a = [1,2,3];
    print("here");
    print(#a); 
    a[1] = a[2];
    print(a[1]); 
    b =  a + [9,10];
    (0 <= i < 5| true) print(b[i]);
}
