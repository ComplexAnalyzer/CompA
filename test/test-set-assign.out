1
2
3
4
4
here
3
3
1
3
3
9
10
